libdnf/repo/Repo.cpp

```diff
@@ -33,12 +33,24 @@
 #define DIGITS "0123456789"
 #define REPOID_CHARS ASCII_LETTERS DIGITS "-_.:"
 
+#define MD_TYPE_PRIMARY "primary"
+#define MD_TYPE_FILELISTS "filelists"
+#define MD_TYPE_PRESTODELTA "prestodelta"
```

Comment:
`deltainfo` and `prestodelta` are the same data, with differing filenames. We need both recognized for the DeltaRPM metadata.